platform/android/export/export.cpp

```diff
@@ -37,6 +37,8 @@
 #include "editor/editor_settings.h"
 #include "editor/export/editor_export.h"
 
+String get_default_android_sdk_path();
```

Comment:
Can you move this method to `EditorPaths` for consistency.

Reply:
I actually think this should be in `platform/android/export/export.cpp` as it's specific to the Android export pipeline and we should aim to keep solutions local.